test/test-spawn.c

```diff
@@ -215,6 +215,54 @@ TEST_IMPL(spawn_fails_check_for_waitpid_cleanup) {
 #endif
 
 
+#ifdef _WIN32
+TEST_IMPL(spawn_requires_elevation) {
+  int r;
+  uv_fs_t req;
+  uv_loop_t* loop = uv_default_loop();
+  char path[1024];
+  size_t path_size = 1024;
+
+  /* Find helper elevation.exe -- Assumed in same location as test runner */
+  r = uv_exepath(exepath, &exepath_size);
+  ASSERT(r == 0);
+  exepath[exepath_size] = '\0';
+  snprintf(path, path_size, "%s\\..\\elevation.exe", exepath);
+  ASSERT(0 == uv_fs_realpath(loop, &req, path, NULL));
+
+  /*
+   * Windows XP and Server 2003 don't support GetFinalPathNameByHandleW()
+   */
+  if (req.result == UV_ENOSYS) {
+    uv_fs_req_cleanup(&req);
+    RETURN_SKIP("realpath is not supported on Windows XP");
+  }
+
+  ASSERT(req.ptr != NULL);
+
+  init_process_options("", fail_cb);
+  options.file = options.args[0] = req.ptr;
+
+  printf("Launching %s\n", (char*) req.ptr);
+  r = uv_spawn(uv_default_loop(), &process, &options);
+  printf("r = %d, %s - %s\n", r, uv_err_name(r), uv_strerror(r));
```

Comment:
hmm okay. I found it useful for diagnostics. If I comment out the mapping, then with the `printf()` calls the test fails:
```
C:\work\node\github\libuv>Debug\run-tests spawn_requires_elevation
not ok 1 - spawn_requires_elevation
# exit code 3
# Output from process `spawn_requires_elevation`:
# Launching C:\work\node\github\libuv\Debug\elevation.exe
# r = -4094, UNKNOWN - unknown error
# Assertion failed in test\test-spawn.c on line 257: r == 0 || r == UV_EACCES

C:\work\node\github\libuv>
```
Compared to (without `printf()`'s):
```
C:\work\node\github\libuv>Debug\run-tests spawn_requires_elevation
not ok 1 - spawn_requires_elevation
# exit code 3
# Output from process `spawn_requires_elevation`:
# Assertion failed in test\test-spawn.c on line 255: r == 0 || r == UV_EACCES

C:\work\node\github\libuv>
```